scripts/gen-auth-api-spec.ts

```diff
@@ -298,6 +298,10 @@ function addEmulatorOperations(openapi3: any): void {
         },
         type: "object",
       },
+      usageMode: {
+        enum: ["DEFAULT", "PASSTHROUGH"],
```

Comment:
```suggestion
        enum: ["USAGE_MODE_UNSPECIFIED", "DEFAULT", "PASSTHROUGH"],
```

And regenerate please. I hate this too, but this makes migration easier later

Reply:
Sounds good, done!